website/content/docs/templates/hcl_templates/functions/contextual/vault.mdx

```diff
@@ -49,11 +49,12 @@ local "foo" {
     expression = vault("secrets/hello", "foo")
     sensitive  = true
 }
+```
 
 The `local` block example accesses the Vault path `secrets/foo` and returns the value
 stored at the key `foo`, storing it as the local variable `local.foo`. However, the output of
 the newly stored local variable will be filtered from the Packer build output, and replaced
-with the value '<sensitive>'. See [Local Values](/docs/templates/hcl_templates/locals) for more details.
+with the value '<sensitive>'. See [Local Values](/packer/docs/templates/hcl_templates/locals) for more details.
```

Comment:
Thanks for this fix with the URI! I notice however that since the `<sensitive>` is encompassed by single quotes, it is still parsed as a HTML tag, and since there's no corresponding terminating tag, it ends with an error at rendering.
I suggest encompassing this in between backticks, this emphases the contents more, and gets rid of this error.

```suggestion
with the value `<sensitive>`. See [Local Values](/packer/docs/templates/hcl_templates/locals) for more details.
```

Reply:
Heyy, thanks for catching this! I've committed the changes.